recipe/meta.yaml

```
@@ -0,0 +1,113 @@
+{% set version = '2.0.5' %}
+{% set posix = 'm2-' if win else '' %}
+
+package:
+  name: r-rcppplanc
+  version: {{ version|replace("-", "_") }}
+
+source:
+  - url:
+      - {{ cran_mirror }}/src/contrib/RcppPlanc_{{ version }}.tar.gz
+      - {{ cran_mirror }}/src/contrib/Archive/RcppPlanc/RcppPlanc_{{ version }}.tar.gz
+    sha256: 6c579ad844f73c8cd03a1824c42f8ddc06c93f67deb3a072a99f633e7a063d4c
+  - path: Makevars       # [not win]
+  - path: Makevars.ucrt  # [win]
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - '*/R.dll'        # [win]
+    - '*/Rblas.dll'    # [win]
+    - '*/Rlapack.dll'  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ compiler('cxx') }}            # [not win]
+    - {{ compiler('m2w64_cxx') }}      # [win]
+    - {{ stdlib("c") }}
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
```

Cross-compiling usually still needs the host to have native dependencies:

```suggestion
    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
    - r-matrix                     # [build_platform != target_platform]
    - r-rcpp                       # [build_platform != target_platform]
    - r-rcpparmadillo              # [build_platform != target_platform]
    - r-rcppprogress               # [build_platform != target_platform]
    - r-hdf5r.extra                # [build_platform != target_platform]
```